modules/axfs/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/arceos-org/arceos/tree/main/modules/axfs"
 documentation = "https://arceos-org.github.io/arceos/axfs/index.html"
 
 [features]
+thread-local = ["axns/thread-local"]
```

Comment:
Remove this feature and enable `axns/thread-local` in `arceos-posix-api/uspace` only.